decaylanguage/dec/dec.py

```diff
@@ -369,26 +372,47 @@ def _add_charge_conjugate_decays(self):
         # At last, create the charge conjugate decays:
         # First, make a (deep) copy of the list of relevant Tree instances.
         # Example:
-        # if mother_names_ccdecays = ['anti-M1', 'anti-M2'],
-        # the relevant Trees are the ones describing the decays of ['M1', 'M2'].
+        # if mother_names_ccdecays = ['anti-M10', 'anti-M2+'],
+        # the relevant Trees are the ones describing the decays of ['M10', 'M2-'].
         dict_cc_names = self.dict_charge_conjugates()
 
-        i = 0
+        name2treepos = {}  # match name -> position in list self._parsed_decays
+        for i, t in enumerate(self._parsed_decays):
+             name2treepos[t.children[0].children[0].value] = i
```

Comment:
Instead of these three lines, you can write a dictionary comprehension:

```python
name2treepos = {t.children[0].children[0].value:i for i, t in enumerate(self._parsed_decays)}
```

Reply:
Good point. Will commit that in a sec ...